gempy/posterior_analysis.py

```diff
@@ -222,7 +246,7 @@ def change_input_data(self, interp_data, i):
         # replace interface data
         interp_data.geo_data_res.interfaces[["X", "Y", "Z"]] = self.input_data[i][0]
         # replace foliation data
-        interp_data.geo_data_res.orientations[["G_x", "G_y", "G_z", "X", "Y", "Z", "dip", "azimuth", "polarity"]] = self.input_data[i][1]
+        interp_data.geo_data_res.orientations[["X", "Y", "Z", "dip", "azimuth", "polarity"]] = self.input_data[i][1]
```

Comment:
Did you delete this on purpose or is one a mismatch between your code and mine?

Reply:
deleted on purpose as in legacy script I never saved gradient values, class will become deprecated in future anyways